src/install/browserFetcher.ts

```diff
@@ -137,7 +137,8 @@ export async function downloadBrowserWithProgressBar(browsersPath: string, brows
       const {error} = await downloadFile(url, zipPath, progress);
       if (!error)
         break;
-      if (attempt < N && error && typeof error === 'object' && typeof error.message === 'string' && error.message.includes('ECONNRESET')) {
+      const errorMessage = typeof error === 'object' && typeof error.message === 'string' ? error.message : '';
```

Comment:
```suggestion
      const errorMessage = String(error);
```
I think thats a valid use-case for the `String` constructor, or what do you think? Since it's include in the end, the prefix `Error: ` should also have no side effects.

Reply:
We only want to do this for well-known node errors and want to disregard everything else. So we'd rather be on a strict side